handler.py

```diff
@@ -129,6 +153,7 @@ def get_recent_logs_handler(event, context):
 
 
 def add_log_entry_handler(event, context):
+    # Authorization handled in the frontend
```

Comment:
I don't think this is technically correct, since the logs are coming from the site. And anyways, since this is an http api, it shouldn't (and can't) rely on clients for anything.

Reply:
A relic from when I mistakenly believed this was the night log/operator message...